stratuslab/java/jar/src/main/java/com/sixsq/slipstream/connector/stratuslab/StratusLabConnector.java

```diff
@@ -163,13 +160,7 @@ protected void validateLaunch(Run run, User user) throws ValidationException {
 		super.validateLaunch(run, user);
 
 		if (run.getCategory() == ModuleCategory.Image) {
-			ImageModule image = ImageModule.load(run.getModuleResourceUrl());
-			validateImageModule(image, user);
-		}
-		if (run.getCategory() == ModuleCategory.Deployment) {
-			for (Node node : DeploymentModule.load(run.getModuleResourceUrl()).getNodes().values()) {
-				validateImageModule(node.getImage(), user);
```

Comment:
I don't see replacement code for this `if` (`run.getCategory() == ModuleCategory.Deployment`). 
Is there a reason the delete this check ?